packages/restapi/src/lib/spaceV2/acceptInvite.ts

```diff
@@ -0,0 +1,261 @@
+            this.setSpaceV2Data((oldData) => {
+                return produce(oldData, (draft) => {
+                    draft.meta.initiator.signal = data;
+                });
+            });
+
+            sendSpaceNotification(
+                {
+                    signer: this.signer,
+                    chainId: this.chainId,
+                    pgpPrivateKey: this.pgpPrivateKey,
+                },
+                {
+                    senderAddress,
+                    recipientAddress,
+                    status: retry
+                        ? VideoCallStatus.RETRY_RECEIVED
+                        : VideoCallStatus.INITIALIZED,
+                    spaceId,
+                    signalData: data,
+                    env: this.env,
+                    callDetails: details,
+                }
+            );
+        });
+
+        peerConnection.on('connect', () => {
+            peerConnection.send(
+                JSON.stringify({
+                    type: 'isVideoOn',
+                    value: this.data.local.video,
+                })
+            );
+            peerConnection.send(
+                JSON.stringify({
+                    type: 'isAudioOn',
+                    value: this.data.local.audio,
+                })
+            );
+
+            // set videoCallInfo state with status connected for the receiver's end
+            this.setSpaceV2Data((oldData) => {
+                return produce(oldData, (draft) => {
+                    const incomingIndex = getIncomingIndexFromAddress(
+                        oldData.incomingPeerStreams,
+                        recipientAddress
+                    );
+                    draft.incomingPeerStreams[incomingIndex].status = VideoCallStatus.CONNECTED;
+                });
+            });
+        });
+
+        peerConnection.on('data', (data: any) => {
+            if (isJSON(data)) {
+                const parsedData = JSON.parse(data);
+
+                if (parsedData.type === 'isVideoOn') {
+                    console.log('IS VIDEO ON', parsedData.value);
+                    this.setSpaceV2Data((oldData) => {
+                        return produce(oldData, (draft) => {
+                            const incomingIndex = getIncomingIndexFromAddress(
+                                oldData.incomingPeerStreams,
+                                recipientAddress
+                            );
+                            draft.incomingPeerStreams[incomingIndex].video = parsedData.value;
+                        });
+                    });
+                }
+
+                if (parsedData.type === 'isAudioOn') {
+                    console.log('IS AUDIO ON', parsedData.value);
+                    this.setSpaceV2Data((oldData) => {
+                        return produce(oldData, (draft) => {
+                            const incomingIndex = getIncomingIndexFromAddress(
+                                oldData.incomingPeerStreams,
```

Comment:
in this also, this isAudoOn event can be fired before stream object and after stream object as well
before -> when connect is fired, then data is sent (stream isn't fired yet)
after -> once stream is successfully flowing, and a user mutes/unmutes the mic